NuGet.Config

```diff
@@ -2,6 +2,13 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 
 <configuration>
+  <packageRestore>
+    <!-- Currently, the repository's version of NuGet.exe and Visual Studio's version
+         fight over the format of project.lock.json because the one in the respository
+         is newer. To prevent that, turn off package restore. -->
```

Comment:
There is no flag to set this. It could be added as a nuget.config setting, but this is only a problem because the lock files are checked in which goes against the current guidance.